shared/rakefiles/xk_util.rake

```diff
@@ -220,4 +220,35 @@ task :revoke_org_admin => [@gcp_creds_file, :configure_extra_tf_vars] do
   end
 end
 
+# This task restores a list of images in snapshots. 
+task :restore_snapshot_from_image_file, [:snapshot_files] => [@gcp_creds_file, :configure_extra_tf_vars] do |taskname, args|
+  require 'csv'
+
+  pv_zones = {}
+  CSV.parse(%x{
+              #{@exekube_cmd} kubectl get pv -o json | jq -r '.items[] | \"\\(.spec.claimRef.name),\\(.metadata.labels.\"failure-domain.beta.kubernetes.io/zone\")\"'
+           }.chomp).each do |line|
+             pv_zones[line[0]] = line[1]
+           end
+
+  snapshot_files = args[:snapshot_files].split ' '
+
+  snapshot_files.each do |snapshot_file|
+    sh "#{@exekube_cmd} sh -c ' \
+      gsutil ls #{snapshot_file}
+    '", verbose: false
+  end
+
+  snapshot_files.each do |snapshot_file|
+    snapshot_name = snapshot_file[/database-storage-couchdb-couchdb-\d-\d+-\d+/, 0]
+    sh "#{@exekube_cmd} sh -c ' \
+    gcloud compute images create image-disk-pv-#{snapshot_name} --source-uri=#{snapshot_file}
```

Comment:
Why is the first newline escaped, but no others?

Reply:
because it is a script of multiple lines with new lines at the end.